web/modules/custom/ct_user/src/Controller/GetUserPatchesController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Drupal\ct_user\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use DateTime;
+use DateTimeZone;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Core\Cache\CacheableJsonResponse;
+use Drupal\Core\Cache\CacheableMetadata;
+
+/**
+ * A controller to get daily patches count for the last 1 year for each user.
+ */
+class GetUserPatchesController extends ControllerBase
+{
+
+    /**
```

Comment:
The indentation is not as per Drupal standards. I presume phpcbf changed this. Can you use phpcbf with Drupal standard?